exonum-java-binding/core/rust/src/storage/proof_map_index.rs

```diff
@@ -38,9 +38,32 @@ use storage::{
 use utils;
 use JniResult;
 
-type Key = [u8; PROOF_MAP_KEY_SIZE];
+#[derive(Clone, Default)]
+struct Key([u8; PROOF_MAP_KEY_SIZE]);
```

Comment:
Doesn't the plain array work any longer?

Reply:
`ObjectHash` trait is required for keys. There is no ability to implement it for type alias due to compiler restrictions